app/controllers/users/passwords_controller.rb

```diff
@@ -7,13 +7,14 @@ def edit
     end
 
     def update
-      @update_user_password_form = UpdateUserPasswordForm.new(current_user)
-
-      result = @update_user_password_form.submit(user_params)
+      @update_user_password = UpdateUserPassword.new(
+        current_user, user_session, user_params[:password]
```

Comment:
I agree that they're clearly named, but without keywords they are order-dependent so you must know what order they have to be in. My rule of thumb these days is to use named args for any class with 2 or more args since I don't see much of a downside...but that's just me!